provider/docker.go

```diff
@@ -614,11 +621,28 @@ func listServices(ctx context.Context, dockerClient client.APIClient) ([]dockerD
 	}
 
 	var dockerDataList []dockerData
+	var dockerDataListTasks []dockerData
 
 	for _, service := range serviceList {
 		dockerData := parseService(service, networkMap)
 
-		dockerDataList = append(dockerDataList, dockerData)
+		sticky, _ := strconv.ParseBool(provider.getSticky(dockerData))
```

Comment:
I think we should remove this line. We don't need to know the stickiness to bypass or not the swarm LB.